snuba/consumers/consumer_builder.py

```diff
@@ -239,6 +239,7 @@ def build_streaming_strategy_factory(
             input_block_size=self.input_block_size,
             output_block_size=self.output_block_size,
             initialize_parallel_transform=setup_sentry,
+            health_check_file="/tmp/health.txt",
```

Comment:
I would not hardcode this sort of thing in the Snuba codebase, and make it configurable. If our ops setup is not generic enough to deal with automatically adding this parameter to all snuba consumers (and configuring health checks appropriately), we should fix that instead.

Reply:
fixed